identity-apps-core/apps/authentication-portal/src/main/webapp/pushAuth.jsp

```diff
@@ -77,10 +77,12 @@
         authenticationFailed = "true";
 
         if (request.getParameter(Constants.AUTH_FAILURE_MSG) != null) {
-            errorMessage = request.getParameter(Constants.AUTH_FAILURE_MSG);
+            String error = request.getParameter(Constants.AUTH_FAILURE_MSG);
 
-            if (errorMessage.equalsIgnoreCase("authentication.fail.message")) {
+            if (error.equalsIgnoreCase("authentication.fail.message")) {
                 errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry.code.invalid");
+            } else if (!error.equalsIgnoreCase(AuthenticationEndpointUtil.i18n(resourceBundle, error))) {
+                errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, error);
             }
```

Comment:
If matching i18 key found.

<img width="977" height="704" alt="Image" src="https://github.com/user-attachments/assets/7e13421a-9a8c-4a63-ab4c-339602040eea" />

If matching i18 key not found.

<img width="884" height="654" alt="Image" src="https://github.com/user-attachments/assets/ceaf1a08-fc61-4d21-9fe9-cdf983a48f31" />